python/sglang/srt/managers/scheduler.py

```diff
@@ -1398,6 +1400,34 @@ def run_scheduler_process(
     dp_rank: Optional[int],
     pipe_writer,
 ):
+    # current process
```

Comment:
wrap this as a unity function in `srt/utils.py`

Reply:
Done as suggestion.